### ISSUE TITLE
miqgrep -i isn't working correctly, it's trying to find files named the regex.

### ISSUE THREAD

**Issue by jeffmcutter** [jeffmcutter/cf_shortcuts#18]
(Using Ansible as transport)

**Comment by jeffmcutter** [comment]
Also doesn't match correctly with pssh transport, but fails silently.

**Comment by jeffmcutter** [comment]
```
SYNOPSIS
       zgrep [ grep_options ] [ -e ] pattern filename...
```
The -e option must be last.

**Comment by jeffmcutter** [comment]
Fixed by 0ab848f724f5d37824e3d387fbd9809da904c1c6.
